src/test/java/uk/gov/hmcts/reform/bulkscanning/service/SearchServiceTest.java

```diff
@@ -150,4 +152,32 @@ public void testRetrieveByDcn() throws Exception {
         SearchResponse searchResponse = paymentService.retrieveByDcn(TEST_DCN_REFERENCE);
         assertThat(searchResponse.getPayments().get(0).getDcnReference()).isEqualTo(TEST_DCN_REFERENCE);
     }
+
+    @Test
+    @Transactional
```

Comment:
why Transactional for unit test?

Reply:
Removing transactional annotation as these test cases do not interact with the database but use a mocked version of results. There are no changes being made in the database to be reverted back.